### PR TITLE
[RFC][NI] Improve demo's style

### PR DESCRIPTION
**Pull request by AndreJoaquim**
NI

### Context
Our demo app's design sucked. Mea culpa. Now it's fixed!

### Screenshots:
screen shot 2017-11-16 at 18 24 24

### Special thanks:
@mmelo @msssequeira @asantos00 @fredcerdeira for helping me out with this one :props: 👊 
@paulomcg thanks for the heart

### CODE REVIEW

**Review by asantos00 — APPROVED** [review]
You're being too humble, **fixed** is not the word for this **awesome job** you did 🙏

File: tests/dummy/app/components/demo-page.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import $ from 'jquery';
+
+const DEVILS_NUMBER = 666 / 2; // Give this a proper name  (╯°□°）╯︵ ┻━┻
```
Comment (telmaantunes) [inline tests/dummy/app/components/demo-page.js:4]: this is meant to stay this way?

File: tests/dummy/app/components/demo-page.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export default Component.extend({
+
+  didRender() {
+    this._super(...arguments);
+
+    $(document).on('click', 'a[href^="#"]', function(event) {
```
Comment (telmaantunes) [inline tests/dummy/app/components/demo-page.js]: lets move all these selectors to a options object? 😛

File: tests/dummy/app/styles/app.scss [hunk 3]
```diff
@@ -1,22 +1,145 @@
...
+  &__current-version {
+    color: #fff;
+    text-align: right;
+    font-size: em(14px);
+    position: absolute;
+    right: 16px;
+    bottom: 16px;
+
+    span {
+      font-weight: 700;
```
Comment (telmaantunes) [inline tests/dummy/app/styles/app.scss:67]: should we be using our font here? and therefore no font-weights?

File: tests/dummy/app/styles/app.scss [hunk 4]
```diff
@@ -1,22 +1,145 @@
...
+        transform: scale3d(0,0,0);
+      }
+    }
+  }
+}
+
+.container {
+  @include column(12);
+  display: flex;
+  height: calc(100vh - 156px);
```
Comment (telmaantunes) [inline tests/dummy/app/styles/app.scss]: what's 156px?

File: tests/dummy/app/styles/app.scss [hunk 5]
```diff
@@ -1,22 +1,145 @@
...
+
+  &--bigger {
+    height: calc(100vh - 64px);
+  }
+
+  .list {
```
Comment (telmaantunes) [inline tests/dummy/app/styles/app.scss]: what's 64px?

File: tests/dummy/app/components/demo-page.js (hunk 2 above)
Comment (mmelo) [reply tests/dummy/app/components/demo-page.js]: ☝️

File: tests/dummy/app/styles/app.scss [hunk 6]
```diff
@@ -50,22 +173,18 @@ body {
 .uni-title {
   display: block;
   margin-bottom: 16px;
+  font-family: 'Open Sans', sans-serif;
```
Comment (telmaantunes) [inline tests/dummy/app/styles/app.scss:180]: 😭

File: tests/dummy/app/components/demo-page.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      event.preventDefault();
+
+      let componentsContainer = $('.components');
+      componentsContainer.animate({
+        scrollTop: componentsContainer.scrollTop() + $($.attr(this, 'href')).position().top
+      }, 500);
```
Comment (mmelo) [inline tests/dummy/app/components/demo-page.js]: `const ANIMATION_DURATION = 500;`

**Review by telmaantunes — COMMENTED** [review]
good job @AndreJoaquim 
good job

**Review by mmelo — APPROVED** [review]
Great Job Jack! 💯 🎉

File: tests/dummy/app/styles/app.scss (hunk 3 above)
Comment (asantos00) [reply tests/dummy/app/styles/app.scss:67]: We don't have our font in ember-cli-uniq as it is paid, i think

File: tests/dummy/app/components/demo-page.js (hunk 1 above)
Comment (asantos00) [reply tests/dummy/app/components/demo-page.js:4]: I think it is 😂

File: tests/dummy/app/styles/app.scss (hunk 3 above)
Comment (telmaantunes) [reply tests/dummy/app/styles/app.scss:67]: 😿

File: tests/dummy/app/components/demo-page.js (hunk 1 above)
Comment (AndreJoaquim) [reply tests/dummy/app/components/demo-page.js:4]: Don't you like easter eggs? :trollface: @telmaantunes

File: tests/dummy/app/components/demo-page.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    });
+
+    $('.components').scroll(function() {
+      if ($('.components').scrollTop() > DEVILS_NUMBER) {
+        $('.demo-app__banner').addClass('demo-app__banner--smaller');
+        $('.container').addClass('container--bigger');
+      } else {
+        $('.demo-app__banner').removeClass('demo-app__banner--smaller');
```
Comment (fredcerdeira) [inline tests/dummy/app/components/demo-page.js]: cache this selectors into vars

**Review by fredcerdeira — APPROVED** [review]

File: tests/dummy/app/components/demo-page.js (hunk 1 above)
Comment (niceoutput) [reply tests/dummy/app/components/demo-page.js:4]: I like easter eggs 👀

File: tests/dummy/app/styles/app.scss [hunk 9]
```diff
@@ -1,22 +1,145 @@
...
+  width: 100%;
+  height: 156px;
+  padding: 32px 0 24px;
+  background-color: color("primary");
+  text-align: center;
+  position: relative;
+  display: flex;
+  z-index: 1;
+
+  transition: height .2s ease-out;
```
Comment (niceoutput) [inline tests/dummy/app/styles/app.scss]: Avoid transition height, use scale instead... performance matters 🚀

**Review by telmaantunes — APPROVED** [review]

**Pull request merged**